tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2010,6 +2105,33 @@ func getFSSizeMb(f *framework.Framework, pod *v1.Pod) (int64, error) {
 	return intSizeInMb, nil
 }
 
+// getFSSizeMbOnSVC returns filesystem size of POD in Mb , which is created on supervisor
+func getFSSizeMbOnSVC(f *framework.Framework, pod *v1.Pod, namespace string) (int64, error) {
```

Comment:
why do we need this new method?

Reply:
In Case of WCP  The command slightly differs , Existing getFSSizeMb() was not working

This is the command I am using in case of WCP     
cmd := []string{"exec", pod.Name, "--namespace=" + namespace, "--", "/bin/sh", "-c", "df -Tkm | grep /mnt/volume1"}
	output := framework.RunKubectlOrDie(namespace, cmd...)
	gomega.Expect(strings.Contains(output, ext4FSType)).NotTo(gomega.BeFalse())

In valilla/GC it works with below code
output, err := storage_utils.PodExec(f, pod, "df -T -m | grep /mnt/volume1")